### ISSUE TITLE
Queen of Pain (QOP)'s Bloodfeather Boots do not have leg texture with Arcana

### ISSUE THREAD

**Issue by real-vaLor** [ValveSoftware/Dota2-Gameplay#9343]
### Cosmetic Item Name

Bloodfeather Boots

### Description

Bloodfeather Boots should have a leg texture. but with arcana equipped the skin only changes the boots.

### Example Match ID

N/A

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106837775/7b1dad5a-fff4-4906-84ad-b66c99998d6b)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106837775/1384a37a-f465-4cde-9a14-995ed8780adf)

**Comment by Avarise831** [comment]
#2019

**Comment by IIaBeJl96** [comment]
> #2019

No, the link to your topic is not about that at all! You didn't even read what it was about!?
But my link about it [6087](https://github.com/ValveSoftware/Dota2-Gameplay/issues/6087)

**Comment by IIaBeJl96** [comment]
> > #2019
> 
> No, the link to your topic is not about that at all! You didn't even read what it was about!? But my link about it [6087](https://github.com/ValveSoftware/Dota2-Gameplay/issues/6087) Dec 21, 2022!!! 
It's been over 6 months and no one cares..
